sites/dotnet-guide/contents/topics/data/index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Data Access
+subtitle: Working with data storage mechanisms
+accent: primary
+icon: fas fa-database
+---
+
+Data access is critical to many line-of-business applications. Whether it is a traditional SQL relational database or the latest in NoSQL storage, JetBrains can help with first-class tools.
+
+## See Also
```

Comment:
Add DataGrip here as the first link? If JetBrains can help with data, DataGrip is the primary actor from which other products benefit.